app/Controllers/configureController.php

```diff
@@ -398,6 +398,8 @@ public function queryAction(): void {
 			if (!empty($params['shareOpml']) && ctype_digit($params['shareOpml'])) {
 				$queryParams['shareOpml'] = (bool)$params['shareOpml'];
 			}
+			unset($queryParams['name']);
+			unset($queryParams['rid']);
```

Comment:
Fine by me, but there seems to be a problem?

Reply:
Fixed